Sources/BraintreeCore/Analytics/BTAnalyticsService.swift

```diff
@@ -1,8 +1,10 @@
 import Foundation
 
-class BTAnalyticsService: Equatable {
+class BTAnalyticsService: AnalyticsSendable {
```

Comment:
Take it or leave it, but I know for other classes we have had the protocol conformance in an extension so we can add a `MARK:` to make it clear where the protocol starts/ends.

Reply:
We could! Since the protocol is just for testing purposes, I thought that would make the functionality of BTAnalyticsService harder to read - but open to whatever folks prefer!